wd_core.au3

```diff
@@ -250,21 +250,19 @@ EndFunc   ;==>_WD_CreateSession
 ; Return values .: Success - 1
 ;                  Failure - 0 and sets @error to $_WD_ERROR_Exception
 ; Author ........: Danp2
-; Modified ......:
+; Modified ......: mLipok
 ; Remarks .......:
 ; Related .......: _WD_CreateSession, _WD_LastHTTPResult
 ; Link ..........: https://www.w3.org/TR/webdriver#delete-session
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_DeleteSession($sSession)
 	Local Const $sFuncName = "_WD_DeleteSession"
-	Local $sResponse = __WD_Delete($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession)
-	Local $iErr = @error
+	__WD_Delete($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession)
+	Local $iErr = ((@error) ? ($_WD_ERROR_Exception) : ($_WD_ERROR_Success))
 
-	If $iErr Then
-		Return SetError(__WD_Error($sFuncName, $_WD_ERROR_Exception, $sResponse), 0, 0)
-	EndIf
-	Return SetError(__WD_Error($sFuncName, $_WD_ERROR_Success, $sResponse), 0, 1)
+	Local $sMessage = (($iErr) ? ('Error occurs when trying to delete session') : ('WebDriver session deleted'))
+	Return SetError(__WD_Error($sFuncName, $iErr, $sMessage), 0, 0)
```

Comment:
here should be 0 or 1
My mistake. 
Do you agree ?

Reply:
Yes, that's how it was prior to this PR. 😉

Reply:
Take a look: [_WD_DeleteSession return 0/1](https://github.com/Danp2/au3WebDriver/pull/290/commits/751d873fe17f50325632075c35f997d22ebc33a0)